tests/mpi_pi.f90

```diff
@@ -67,7 +67,7 @@ program calculatePi
 
   if (worldRank == 0) then
     call date_and_time(VALUES=timeValues)
-    write(*,"(A,I2,A,I2,A,I2.2)") "End Time = ", timeValues(5), ":", timeValues(6), ":", timeValues(7)
+    write(*,"(A,I2,A,I2,A,I2.2)") "End Time = ", timeValues(1), ":", timeValues(2), ":", timeValues(3), ":", timeValues(5), ":", timeValues(6), ":", timeValues(7)
```

Comment:
Currently this is what the output from the tests look like for the End Time:
`End Time = **: 1:04
:17:27:02`

The year shows up in `**` format, and the tests pass correctly.